### DISCUSSION TITLE
Is it possible to display groups list in a plugin instead of NetBox admin ?

### DISCUSSION THREAD

**Discussion by sysdagenaim** [netbox-community/netbox#11773] [Q&A]
Is it possible to display groups list in a plugin instead of NetBox admin ?  
I always get the error " 'QuerySet' object has no attribute 'restrict'"

What shall I do ?

**Comment by kkthxbye-code** [comment]
The nebox ObjectListView uses the ObjectPermissionRequiredMixin to manage permissions which calls restrict on the queryset. You can try overriding the `has_permission` method from ObjectPermissionRequiredMixin, but not quite sure if that works.
